apps/aechannel/src/aesc_chain_watcher.erl

```diff
@@ -204,6 +205,16 @@ start_link(Type, TxHash, ChanId, MinDepth, Mod) ->
                                      requests => Reqs},
                           ?GEN_SERVER_OPTS).
 
+restart_watcher(ChanId, Mod) ->
```

Comment:
So this is exactly like `start_link/4` without the initial transaction to be tracked?

Reply:
How about making this `start_link/2`?